core/src/main/java/org/apache/accumulo/core/iterators/SortedMapIterator.java

```diff
@@ -95,6 +111,9 @@ public void next() throws IOException {
   public void seek(Range range, Collection<ByteSequence> columnFamilies, boolean inclusive)
       throws IOException {
 
+    if (interruptFlag != null && interruptFlag.get())
```

Comment:
`interruptFlag` should never be null. We can require non-null anywhere we accept it.